core/storage/story/gae_models.py

```diff
@@ -48,12 +48,13 @@ class StoryModel(base_models.VersionedModel):
     description = ndb.StringProperty(indexed=False)
     # A set of notes, that describe the characters, main storyline, and setting.
     notes = ndb.TextProperty(indexed=False)
-    # The ISO 639-1 code for the language this question is written in.
+    # The ISO 639-1 code for the language this story is written in.
     language_code = ndb.StringProperty(required=True, indexed=True)
     # The schema version for the story_contents.
     schema_version = (
         ndb.IntegerProperty(required=True, default=1, indexed=True))
-    # The story graph specifying the connection between nodes.
+    # The story contents dict specifying the list of story nodes and the
+    # connection between them. Modeled by class StoryContents.
```

Comment:
`Modelled by class StoryContents (see story_domain.py for its current schema).`

Reply:
Done.